src/entities/User.ts

```diff
@@ -14,133 +15,110 @@ import Matching from './Matching';
 @Entity('pear_user')
 class User {
   @PrimaryGeneratedColumn('uuid')
-  id: string;
+  uuid: string;
 
-  /** User's clubs */
-  @ManyToMany(
-    type => Club,
-    club => club.users
-  )
-  clubs: Club[];
-
-  /** User's facebook profile link */
-  @Column({
-    type: 'varchar',
-    nullable: true,
-  })
-  facebook: string | null;
+  /** Google ID of user */
+  @Column({ type: 'varchar' })
+  googleID: string;
 
   /** User first name */
-  @Column({
-    type: 'varchar',
-  })
+  @Column({ type: 'varchar' })
   firstName: string;
 
-  /** Google ID of user */
-  @Column({
-    type: 'varchar',
-  })
-  googleID: string;
+  /** User last name */
+  @Column({ type: 'varchar' })
+  lastName: string;
+
+  /** Net ID of user */
+  @Column({ type: 'varchar', unique: true })
+  netID: string;
+
+  /** User's pronouns */
+  @Column({ type: 'varchar', nullable: true })
+  pronouns: string | null;
 
   /** User's graduation year */
-  @Column({
-    type: 'varchar',
-    nullable: true,
-  })
+  @Column({ type: 'varchar', nullable: true })
   graduationYear: string | null;
 
+  /** User's major */
+  @ManyToOne(type => CornellMajor, major => major.users, { nullable: true, onDelete: "CASCADE" })
```

Comment:
any reason for double quotes instead of single here?

Reply:
This also gets changed in the next PR!